assets/cluster-dns-cr.yaml

```diff
@@ -2,8 +2,7 @@
 apiVersion: dns.openshift.io/v1alpha1
 kind: ClusterDNS
 metadata:
-  name: default
-  namespace: openshift-dns-operator
+  name: openshift-dns
```

Comment:
Is there a reason to change the name from "default"? The resource kind and API group are sufficient to describe what it is, and I see other operators tend to use "default" or "instance".  For example, [cluster-network-operator ignores, and logs that it ignores, any networkconfig that is not named "default"](https://github.com/openshift/cluster-network-operator/blob/master@%7B2018-12-19%7D/pkg/controller/networkconfig/networkconfig_controller.go#L85-L89), and [cluster-openshift-apiserver-operator only looks for openshiftapiserveroperatorconfigs named "instance"](https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master@%7B2018-12-19%7D/pkg/operator/workload_controller.go#L103).  We also use the name "default" for [the default clusteringress](https://github.com/openshift/cluster-ingress-operator/blob/master@%7B2018-12-19%7D/assets/defaults/cluster-ingress.yaml#L6).

Reply:
Changing this has ripple effects, let's not change it in this PR